src/entities/lab-Staff.entity.ts

```diff
@@ -0,0 +1,58 @@
+import {
+    Entity,
+    PrimaryGeneratedColumn,
+    Column,
+    CreateDateColumn,
+    UpdateDateColumn,
+    DeleteDateColumn,
+    OneToMany,
+} from "typeorm";
+
+import { TestResult } from "@/entities"
+
+import { Gender, LabStaffRole } from "@/shared"
+
+@Entity( { name: "lab_staff" } )
+export class LabStaff {
+    @PrimaryGeneratedColumn ("uuid")
+    id! : string;
+
+    @Column({ type: "varchar", name: "first_name", nullable: false })
+    firstName!: string;
+
+    @Column({ type: "varchar", name: "last_name", nullable: false })
+    lastName!: string;
+
+    @Column({ type: "varchar", nullable: false })
+    email!: string;
+
+    @Column({ type: "integer", name: "phone_number", nullable: true })
+    phoneNumber!: number | null;
+
+    @Column( { type: "varchar", nullable: true } )
+    address!: string | null;
+
+    @Column ( { type: "enum", enum: Gender, default: Gender.OTHER, nullable: true } )
```

Comment:
remove default value. either a value is provided or its null. you wan dey give pple default gender?